cluster/manifests/stackset-controller/01-stack-crd.yaml

```diff
@@ -357,6 +357,20 @@ spec:
                   description: ConfigurationResourcesSpec makes it possible to defined
                     the config resources to be created
                   properties:
+{{- if eq .Cluster.ConfigItems.stackset_inline_configmap_support_enabled "true" }}
+                    configMap:
+                      description: ConfigMap to be owned by Stack
+                      properties:
+                        data:
+                          additionalProperties:
+                            type: string
+                          description: Data of the ConfigMap
+                          type: object
+                        name:
+                          description: Name of the ConfigMap
+                          type: string
+                      type: object
+{{ end }}
```

Comment:
Looks like these were not added to the stack CRD https://github.com/zalando-incubator/stackset-controller/blob/master/docs/stack_crd.yaml

Reply:
Noticed while working on https://github.com/zalando-incubator/kubernetes-on-aws/pull/7297

Reply:
Thanks. Yes, this wasn't merged 🤦  I created a separate PR to fix it: https://github.com/zalando-incubator/stackset-controller/pull/621

Reply:
We merged it to master now. It's slightly different that here (code comments) but the structure is the same. Let's update this with one of the subsequent PRs that modify the spec.

Reply:
Thank you, I can update that within https://github.com/zalando-incubator/kubernetes-on-aws/pull/7297